asciidoctorj-core/src/main/java/org/asciidoctor/jruby/extension/processorproxies/AbstractProcessorProxy.java

```diff
@@ -217,7 +217,7 @@ private static void handlePositionalAttributesAnnotation(Class<? extends Process
                 positionalAttrs.add(positionalAttribute);
             }
             rubyClass.callMethod(rubyRuntime.getCurrentContext(), "option", new IRubyObject[]{
-                    rubyRuntime.newSymbol("pos_attrs"),
+                    rubyRuntime.newSymbol("positional_attrs"),
```

Comment:
I'm going to go ahead a remap this to avoid breaking existing extensions. But since AsciidoctorJ bundles a specific version of Asciidoctor core, you can just fix it directly here.